docs/releasenotes/7.0.0.rst

```diff
@@ -58,6 +58,20 @@ and :py:meth:`~PIL.ImageOps.fit`` functions.
 See :ref:`concept-filters` to learn the difference. In short,
 ``Image.NEAREST`` is a very fast filter, but simple and low-quality.
 
+Image.draft() return value
```

Comment:
@radarhere Does this description look good for you?

Reply:
I've created https://github.com/uploadcare/pillow-simd/pull/60 with my suggestion.